lib/services/datasets.js

```diff
@@ -4,7 +4,9 @@ var invariant = require('../../vendor/invariant');
 var hat = require('../../vendor/hat');
 var makeService = require('../make_service');
 
-var Datasets = module.exports = makeService('MapboxDatasets');
+/**
+ */
```

Comment:
maybe add a `@class` tag so its clear in the future why these empty comment blocks are here 🙃